src/styles/global.scss

```diff
@@ -28,14 +28,41 @@
 /**
 Some USWDS components have pseudo elements (::before and ::after) that use the mask CSS property. This property does not always render in Chrome 108 (seems to work in FF  and Safari)
 */
+
+// This control the icon in the alerts
 .usa-alert--info:before {
     -webkit-mask: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTIgMkM2LjQ4IDIgMiA2LjQ4IDIgMTJzNC40OCAxMCAxMCAxMCAxMC00LjQ4IDEwLTEwUzE3LjUyIDIgMTIgMnptMSAxNWgtMnYtNmgydjZ6bTAtOGgtMlY3aDJ2MnoiLz48L3N2Zz4=) no-repeat center/contain;
 }
 
+// This sets the down arrow in the gov banner
 .usa-banner__button::after {
     -webkit-mask: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTYuNTkgOC41OSAxMiAxMy4xNyA3LjQxIDguNTkgNiAxMGw2IDYgNi02eiIvPjwvc3ZnPg==) no-repeat center/contain;
 }
 
+// This sets the up arrow in the gov banner
+.usa-banner__button[aria-expanded=true]::after{
```

Comment:
did your gov arrow dropdown just work? I've had to place all these overrides in

Reply:
it... did? I have no idea why it would or wouldn't tho.

Reply:
I think USWDS is using the CSS mask property which isn't supported by all browsers. However I haven't figured out why this is only coming up in this repo.